src/ConductorSharp.Toolkit/Service/ScaffoldingService.cs

```diff
@@ -0,0 +1,286 @@
+                            break;
+                        case "select":
+                            var optionsToken = param.Value.SelectToken("options");
+
+                            string[] options = null;
+
+                            if (optionsToken is JArray optionsArray)
+                            {
+                                options = optionsArray.ToObject<string[]>();
+                            }
+                            if (options != null && options.Length > 0)
+                            {
+                                inputLinesBuilder.AppendLine($"        /// <remark>");
+                                inputLinesBuilder.AppendLine($"        /// Options: {string.Join(',', options)}");
+                                inputLinesBuilder.AppendLine($"        /// </remark>");
+                            }
+                            inputLinesBuilder.AppendLine($"        [JsonProperty(\"{param.Key}\")]");
+                            LangUtils.MakeValidMemberName(SnakeCaseUtil.ToPascalCase(param.Key), "A", out propertyName);
+                            inputLinesBuilder.AppendLine($"        public dynamic {propertyName} {{ get; set; }}");
+                            break;
+                        default:
+                            break;
+                    }
+                }
+            }
+            catch (Exception)
+            {
+                if (workflowDefinition.InputParametersJSON != null)
+                    Console.WriteLine(
+                        $"Unable to parse input parameters {string.Join(',', workflowDefinition.InputParametersJSON)} for {workflowDefinition.Name})"
+                    );
+                else
+                    Console.WriteLine($"No input parameters defined for {workflowDefinition.Name}");
+            }
+
+            if (string.IsNullOrEmpty(workflowDefinition.OwnerEmail))
+                logger.LogWarning($"No owner email defined for task {workflowDefinition.Name}");
+
+            if (string.IsNullOrEmpty(workflowDefinition.OwnerApp))
+                logger.LogWarning($"No owner app defined for task {workflowDefinition.Name}");
+
+            lines = lines.Replace("{{inputProperties}}", inputLinesBuilder.ToString());
+            lines = lines.Replace("{{workflowName}}", name);
+            lines = lines.Replace("{{note}}", note);
+            lines = lines.Replace("{{ownerEmail}}", $"{workflowDefinition.OwnerEmail}");
+            lines = lines.Replace("{{ownerApp}}", $"{workflowDefinition.OwnerApp}");
+            lines = lines.Replace("{{originalName}}", $"{workflowDefinition.Name}");
+            lines = lines.Replace("{{description}}", workflowDescription);
+            lines = lines.Replace("{{commentDescription}}", workflowDescription?.Replace('\n', ','));
+
+            if (config.Dryrun)
+                return null;
+
+            return lines;
+        }
+        public async Task<string> CreateTaskClass(TaskDefinition taskDefinition)

```

Comment:
As for previous method, we can drop the async Task<...>

Reply:
Done